src/main/java/Main.java

```diff
@@ -22,6 +22,7 @@ public static void main(String[] args) {
         SwingUtilities.invokeLater(new Runnable() {
             @Override
             public void run() {
+                ReceiptSearchForm.main(new String[] {});
                 new DrugRequestFrame();
             }
```

Comment:
I think it's good you added your ReceiptSearchForm.main inside the SwingUtilities.invoke later, this will allow us to catch certain cases where it will not run. However one issue i have is that your searchForm is currently being called first from the  main. So its kind of weird where the search frame is generating first, then my drug request frame will generate after. 

I think your should make another SwingUtilities.invokelater below mine and add your receipt search form there, afterwards we will figure out how and when to call your Receipt search form cool?